src/components/ebay-chip/chip.stories.js

```diff
@@ -0,0 +1,67 @@
+import { tagToString } from "../../../.storybook/storybook-code-source";
+import { addRenderBodies } from "../../../.storybook/utils";
+import Readme from "./README.md";
+import Component from "./index.marko";
+
+const Template = (args) => ({
+    input: addRenderBodies(args),
+});
+
+export default {
+    title: "building blocks/ebay-chip",
+    component: Component,
+    parameters: {
+        docs: {
+            description: {
+                component: Readme,
+            },
+        },
+    },
+
+    argTypes: {
+        renderBody: {
+            control: { type: "text" },
+            description: "Text to be displayed in the chip",
+        },
+        deleteButtonLabel: {
+            control: { type: "text" },
+            description:
+                "A11y text for the delete button, also determines if delete button is shown",
+        },
+        onDelete: {
+            action: "on-delete",
+            description: "Triggered when delete button is clicked",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ el, checked, originalEvent }",
```

Comment:
To what is `checked` referring? Should it be `deleted` instead?

Reply:
Good catch, this was actually completely wrong, must've been a copy-paste error. I fixed it.